addons/web/static/src/views/fields/many2one/many2one_field.js

```diff
@@ -106,7 +106,7 @@ export class Many2OneField extends Component {
         this.addDialog = useOwnedDialogs();
 
         this.focusInput = () => {
-            this.autocompleteContainerRef.el.querySelector("input").focus();
+            this.autocompleteContainerRef.el?.querySelector("input").focus();
```

Comment:
Why do you need this change ?

Reply:
This is also a bug in master so I was thinking of to do the fix in the same PR
https://www.awesomescreenshot.com/video/25272394?key=9480fb6d76342c5ea28a9e2ee8cca557

1. Go to an app, for example, CRM;
2. Click the log note in chatter and open the full composer;
3. Load a template and preview the change and it produce the traceback;

I think the problem with the code is that you cannot guarantee the `this.autocompleteContainerRef.el` is always there in the dialog especially when there are multiple dialogs overlapping and some of them will be closed.

Reply:
And since we might drop this PR. Maybe it's better to do the fix in the master with a separate PR?

Reply:
Yes it s a good idea to do another PR with your fix and a test :) Thx

Reply:
The bug isn't in v17 ?